src/controllers/orderController.js

```diff
@@ -5,58 +5,100 @@ const { BadRequestError } = require('../errors');
 const PurchasedAlbum = require('../models/PurchasedAlbum');
 const CustomError = require('../errors');
 const { checkPermissions } = require('../utils');
+const { sendOrderCompletedEmail } = require('../mailing/sender');
+const mongoose = require('mongoose');
+
+// Connect to the MongoDB database
+mongoose.connect(process.env.MONGO_URL, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+});
+const db = mongoose.connection;
+
+db.on('error', (error) => {
+  console.error('Database connection error:', error);
+});
+
+db.once('open', () => {
+  console.log('Connected to the database');
+});
```

Comment:
I'm not clear on why this is needed here?

Reply:
@akosasante I was trying to implement a transaction and got some error and after attempts to find a solution included this mongo connection